packages/block-library/src/post-comments-form/block.json

```diff
@@ -22,5 +22,5 @@
 			"lineHeight": true
 		}
 	},
-	"style": "wp-block-post-comments-form"
+	"style": [ "wp-block-post-comments-form", "wp-block-buttons", "wp-block-button" ]
```

Comment:
I noticed one limitation in the `block_type_metadata` filter callback:

```php
wp_enqueue_block_style( $metadata['name'], array( 'handle' => $handle ) );
```

When using an array format you need to pass style handles. It doesn't support file paths. Should we fix it?

Reply:
Fair point, I forgot that we allow paths.
Fixed in https://github.com/WordPress/gutenberg/pull/32510/commits/2c2edf0bd8aad356ebf6c6e55b502267d7ca27bd 👍